dandiapi/api/tests/test_version.py

```diff
@@ -878,3 +885,26 @@ def test_version_rest_delete_draft_admin(api_client, admin_user, draft_version):
     assert response.status_code == 403
     assert response.data == 'Cannot delete draft versions'
     assert draft_version in Version.objects.all()
+
+
+@pytest.mark.django_db
+def test_version_no_metadata_updates(api_client: APIClient, admin_user, draft_version: Version):
```

Comment:
I think this should be renamed to something like
```
test_version_rest_update_no_changed_metadata
```

to keep in line with the other metadata update tests, as well as moved up in the file to live with them.